src/content/1.7/modules/concepts/hooks/list-of-hooks.md

```diff
@@ -2135,4 +2135,152 @@ action<KpiIdentifier>KpiRowModifier
     ),
     ```
 
-{{% /funcdef %}}
+action&lt;HookName>Form
+:
+<!--     todo: options form available since ?-->
```

Comment:
@matks do you know when the options form hooks were introduced - in which PrestaShop version?

Reply:
1.7.4 https://github.com/PrestaShop/PrestaShop/pull/8279